src/elements/content-sidebar/activity-feed/comment/Replies.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+import * as React from 'react';
+import noop from 'lodash/noop';
+
+import BaseComment from './BaseComment';
+import LoadingIndicator from '../../../../components/loading-indicator';
+
+import type { GetAvatarUrlCallback, GetProfileUrlCallback } from '../../../common/flowTypes';
+import type { Translations } from '../../flowTypes';
+import type { SelectorItems, User } from '../../../../common/types/core';
+import type { BoxCommentPermission, Comment as CommentType } from '../../../../common/types/feed';
+
+import './Replies.scss';
+
+type ReplyProps = {
+    hasReplies?: boolean,
+    isRepliesLoading?: boolean,
+    onReplySelect: (isSelected: boolean) => void,
+    replies?: CommentType[],
+};
+
+type Props = ReplyProps & {
+    currentUser?: User,
+    getAvatarUrl: GetAvatarUrlCallback,
+    getMentionWithQuery?: (searchStr: string) => void,
+    getUserProfileUrl?: GetProfileUrlCallback,
+    mentionSelectorContacts?: SelectorItems<>,
+    translations?: Translations,
+};
+
+const Replies = ({
+    currentUser,
+    getAvatarUrl,
+    getMentionWithQuery,
+    getUserProfileUrl,
+    hasReplies = false,
+    isRepliesLoading = false,
+    mentionSelectorContacts,
+    onReplySelect,
+    replies = [],
+    translations,
+}: Props) => {
+    const getReplyPermissions = (reply: CommentType): BoxCommentPermission => {
+        const { permissions: { can_delete = false, can_edit = false, can_resolve = false } = {} } = reply;
+        return {
+            can_delete,
+            can_edit,
+            can_resolve,
+        };
+    };
+
+    if (!hasReplies) {
+        return null;
+    }
+
+    return (
+        <div className="bcs-Replies">
+            <div className="bcs-Replies-content">
+                {isRepliesLoading && (
+                    <div className="bcs-Replies-loading" data-testid="replies-loading">
+                        <LoadingIndicator />
+                    </div>
+                )}
+                <ol className="bcs-Replies-list">
```

Comment:
So if this component is used outside of ActivityFeed the ol will actually have list style which is not what we want. If we are going to use the ol here let's specify that style in bcs-Replies-list or use a ul instead. Here's the style for reference https://github.com/box/box-ui-elements/blob/master/src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss#L22